packages/clay-dataset-display/src/__tests__/__snapshots__/ClayDatasetDisplay.js.snap

```diff
@@ -1,11258 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`ClayDatasetDisplay should render a ClayDatasetDisplay and change view on management toolbar view type click 1`] = `
```

Comment:
@matuzalemsteles because we changed a lot of API and how to use the management toolbar and it would be a lot of time consuming to rewrite dataset display logic to fit with it, knowing that this component is not going to be use. So the best think to keep going is just forget about the component, but instead removing it just get rid of the tests to avoid failures

Reply:
Oh yeah, maybe we should add a warning to `README.md` that this component can be deprecated.